lib/pipelines/complexity.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var path = require('path');
+var plato = require('es6-plato');
+var through = require('through2');
+
+module.exports.plato = function(options) {
+	var name;
+
+	try {
+		name = require(path.join(process.cwd(), 'package.json')).name;
+	}
+	catch (e) {}
+
+	var files = [];
+	var platoArgs = {
+		complexity: {},
+		title: name || 'Metal'
+	};
+
+	return through.obj(function(file, encoding, callback) {
+		files.push(file.path);
+		callback();
+	}, function(callback) {
+		var stream = this;
+
+		plato.inspect(files, options.dest, platoArgs, function() {
```

Comment:
That's what I originally had, but the `plato.inspect` method doesn't respect an array of globs the same way `gulp.src` does, it makes it difficult to exclude certain files and would make it inconsistent with how we pass globs to the lint and format tasks.